tests/test_transformations/test_get_cluster_info.py

```diff
@@ -41,6 +43,26 @@ def test_specific_features_with_percentage(self):
         self.assertIn("mean_Gene2", result.columns)
         self.assertNotIn("mean_Gene3", result.columns)
 
+    def test_layer_specific_features_with_percentage(self):
+        # Test functionality with a specific layer
+        result = get_cluster_info(self.adata, layer="scaled")
+        self.assertIsInstance(result, pd.DataFrame)
```

Comment:
@fangliu117 , as this unit test is checking the functionality that a layer specific metric is calculated correctly, please edit it to: 
- No make the assertion that other tests already checked (e..g, result is pd.DataFrame, Cluster is in columns, etc.)
- Add an assertion with known values for the added layer as you did in test_default_features_with_known_values_and_percentage